scripts/print_language_list.py

```
@@ -0,0 +1,14 @@
+"""
+Prints the list of supported languages, for use in the project.yml template
+"""
+
+from solidlsp.ls_config import Language
+
+if __name__ == "__main__":
+    lang_strings = sorted([l.value for l in Language])
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)